src/main/java/slimeknights/tconstruct/library/materials/client/MaterialRenderInfo.java

```diff
@@ -0,0 +1,26 @@
+package slimeknights.tconstruct.library.materials.client;
+
+import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+import net.minecraft.util.ResourceLocation;
+
+/**
+ * Determines the type of texture used for rendering a specific material
+ */
+public interface MaterialRenderInfo {
```

Comment:
IMaterialRenderInfo?

Reply:
Dang. I hate how minecraft/forge uses the IFoobar interface notation.. but I want to stay consistent... thanks